netbox/templates/inc/table_controls_htmx.html

```diff
@@ -2,31 +2,21 @@
 
 <div class="row mb-3 justify-content-between">
   <div class="table-controls noprint col col-12 col-md-8 col-lg-4">
-    <div class="input-group input-group-sm">
-      <input
-        type="text"
-        name="q"
-        class="form-control"
-        placeholder="Quick search"
-        hx-get="{{ request.full_path }}"
-        hx-target="#object_list"
-        hx-trigger="keyup changed delay:500ms"
-      />
+    <div class="input-group input-group-sm quicksearch hide-last-child">
+      <input type="search" results=5 name="q" id="quicksearch" class="form-control" placeholder="Quick search"
```

Comment:
Is `results=5` needed?

Reply:
Switching the input field type to search enables HTML5 functionalities. 
One of those functionalities is to present the user a drop-down of recent searches, results limits this field to a fixed number of recent searches.
[results documentation](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/search#results)

To be honest it's not really needed but makes the UI a bit easier to work with on certain browsers such as Safari (Computers and Mobiles)

Reply:
TIL, thanks!